components/Avatar.tsx

```diff
@@ -82,7 +82,7 @@ const Avatar = ({
       src = defaultImage.ANONYMOUS;
     } else if (collective.isGuest && shouldUseDefaultGuestAvatar(collective.name)) {
       src = defaultImage.GUEST;
-    } else if (type === 'VENDOR') {
+    } else if (type === 'VENDOR' && !collective.imageUrl) {
```

Comment:
Isn't `imageUrl` always set? I don't think this field returns `null` if the account has no uploaded image https://github.com/opencollective/opencollective-api/blob/debt/remove-create-pledge/server/graphql/v2/interface/Account.ts#L802

Reply:
We're not really allowing users to customize the image for newly created vendors, in this context, what happens if the `collective.image` is empty? Well, what role does the `collective.image` even plays?

Reply:
`collective.imageUrl` resolver uses `getImageUrl` which calls `getCollectiveAvatarUrl` https://github.com/opencollective/opencollective-api/blob/debt/remove-create-pledge/server/lib/collectivelib.ts#L38. This function always returns a string, never null, even if there's no image. `collective.image` is included as the hash in the URL, if available.